packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.html

```diff
@@ -0,0 +1,28 @@
+<div data-ui-view>
```

Comment:
**nitpick:** Make this as a named view. And let the view inside the tabs be a default one.

Reply:
Thanks for this Anoop.
As we discuss we will take care in related task.
A comment has been added into the right Jira Story.